ckanext/announcements/templates/admin/announcements.html

```diff
@@ -11,7 +11,7 @@ <h2>{{ _('Announcements to all users') }}</h2>
     role="button"
     class="btn btn-primary"
     title="Schedule new message"
-    data-toggle="modal"
+    data-bs-toggle="modal"
```

Comment:
If you are planning on supporting CKAN 2.9 (or CKAN 2.10 running on Bootstrap 3) you need to keep both attributes the new and the old version